src/api.rs

```diff
@@ -509,6 +543,7 @@ impl Config {
   }
 }
 
+/// Contain all the context configurations
```

Comment:
Omit it or expand it.

Reply:
Omitting it as the function has explanation below